tests/integration_tests/modules/test_combined.py

```diff
@@ -323,7 +329,12 @@ def test_combined_cloud_config_json(
             "/run/cloud-init/combined-cloud-config.json"
         )
         data = json.loads(combined_json)
-        assert data["features"] == get_features()
+        expected_features = get_features()
+        if CURRENT_RELEASE in (FOCAL, JAMMY, MANTIC):
```

Comment:
Nit: You can do `if CURRENT_RELEASE < NOBLE:` here, but it looks like test is still failing.